src/components/progress-tracker/progress-tracker.style.ts

```diff
@@ -111,16 +111,16 @@ const StyledValuesLabel = styled.span<
 `;
 
 const InnerBar = styled.span<
-  Pick<ProgressTrackerProps, "progress" | "size" | "length" | "error">
+  Pick<ProgressTrackerProps, "progress" | "size" | "error">
```

Comment:
Comment (non-blocking): by extending the set of the `ProgressTrackerProps` interface here you're actually saying `progress` can potentially be undefined. It might be worth making it `Required` in this interface as it will never not be a number given it's defaulted in the component prop destructuring and updating this type would mean it would complain if anyone removed that default inadvertently as in that scenario it could end being `undefined%` in the styles. The same is true for `size` and `error` as they have values by the time they are passed to the styled-component we could do the following:
```
Required<Pick<ProgressTrackerProps, "progress" | "size" | "error">>